### PR TITLE
feat(Community Airdrops): Dynamic fee calculation in the airdrop form

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Provides "Fee" section in the airdrop form where fees are presented whenever the from is changed and fully filled with valid input ("What", "To" sections).

Closes: #10547

### Affected areas
`CommunityNewAirdropView`

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

[Screencast from 14.06.2023 17:35:57.webm](https://github.com/status-im/status-desktop/assets/20650004/b5f4cdd0-9025-455e-aa15-ccf546e2bb31)

### CODE REVIEW

**Review by caybro — APPROVED** [review]

File: storybook/pages/CommunityNewAirdropViewPage.qml
Comment (noeliaSD) [inline storybook/pages/CommunityNewAirdropViewPage.qml:1]: Could it be possible to add some editor to easily test the trigger of each type of error?

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml
Comment (noeliaSD) [inline ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml:1]: Maybe I'm missing something but I cannot see these two errors implementations:
- Asset total exceeds the balance remaining in the contract
- Transaction fees exceed block gas limit

Are they something pending?

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml [hunk 1]
```diff
@@ -105,6 +196,55 @@ StatusScrollView {
                 accountName: modelItem.accountName
             }
         }
+
+        function rebuildFeesModel() {
+            feesModel.clear()
+
+            const airdropTokens = ModelUtils.modelToArray(
+                                    root.selectedHoldingsModel,
+                                    ["contractUniqueKey", "accountName",
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml:205]: Are `contractUniqueKey` and `key` redundant? I guess we should need only `key` to identify the item, where the **key** is in this case the **contract unique identifier** data, and then keep the `symbol` information as `symbol` not as `key`.

```suggestion
                                    ["key", "accountName","symbol", "amount", "tokenText",
                                     "networkText"]
```

**Review by noeliaSD — COMMENTED** [review]
In general LGTM!! I didn't test it, just did an insepection in `storybook`.

Added some questions!

File: storybook/pages/CommunityNewAirdropViewPage.qml
Comment (micieslak) [reply storybook/pages/CommunityNewAirdropViewPage.qml:1]: Done :)
Screenshot from 2023-06-15 12-11-58

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml
Comment (micieslak) [reply ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml:1]: The first one you mention is probably this:
Screenshot from 2023-06-15 12-12-35
The second one is not supported at all currently. I've added a new task for that: https://github.com/status-im/status-desktop/issues/11104

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml (hunk 1 above)
Comment (micieslak) [reply ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml:207]: Good point, I improved that a bit, but probably it's the subject for discussion with the backend guys at some point.

Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml:207]: Thank you! Next week we can meet and talk about that bc I think we need to unify things!!

File: storybook/pages/CommunityNewAirdropViewPage.qml
Comment (noeliaSD) [reply storybook/pages/CommunityNewAirdropViewPage.qml:1]: Thanks for adding it!!! :)

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml:1]: Ok!! Thanks for creating the task!

**Review by noeliaSD — APPROVED** [review]

**Pull request merged**